concent_api/concent_api/system_check.py

```diff
@@ -539,3 +579,29 @@ def check_verifier_download_chunk_size(app_configs=None, **kwargs):  # pylint: d
             )]
 
     return []
+
+
+@register()
+def check_middleman_address(app_configs=None, **kwargs):  # pylint: disable=unused-argument
+    if not hasattr(settings, 'MIDDLEMAN_ADDRESS') and 'middleman' in settings.CONCENT_FEATURES:
+        return [create_error_43_middleman_address_is_not_set()]
+
+    if hasattr(settings, 'MIDDLEMAN_ADDRESS'):
+        if not isinstance(settings.MIDDLEMAN_ADDRESS, str):
+            return [create_error_44_middleman_address_has_wrong_type(settings.MIDDLEMAN_ADDRESS)]
+
+    return []
+
+
+@register()
+def check_middleman_port(app_configs=None, **kwargs):  # pylint: disable=unused-argument
+    if not hasattr(settings, 'MIDDLEMAN_PORT') and 'middleman' in settings.CONCENT_FEATURES:
+        return [create_error_45_middleman_port_is_not_set()]
+
+    if hasattr(settings, 'MIDDLEMAN_PORT'):
+        if not isinstance(settings.MIDDLEMAN_PORT, int):
+            return [create_error_46_middleman_port_has_wrong_type(settings.MIDDLEMAN_PORT)]
+        if not 0 < settings.MIDDLEMAN_PORT < 65535:
```

Comment:
We do not allow to use port number 0 or 65535, right?

Reply:
Right.